src/RenderView.tsx

```diff
@@ -22,6 +22,40 @@ interface Props {
 export default function RenderView(props: Props) {
 	const width = 800;
 	const height = 450;
+	
+	const forceMeshesComp = useMemo(() => {
+		const mesh1 = new THREE.Mesh(new THREE.ConeGeometry(.02,.2,8),new THREE.MeshBasicMaterial());
+		const mesh2 = new THREE.Mesh(new THREE.ConeGeometry(.02,.2,8),new THREE.MeshBasicMaterial());
+		mesh1.visible = false;
+		mesh2.visible = false;
+		return [mesh1,mesh2];
+	 }, []);
+	
+	/* Position force velocity for the given frame (in props) */
+	 useEffect(() => {
+		const frameData = props.forceData.frames[props.frame]; //select a single frame
+		if (!frameData) return; //animation doesn't depend on forceFileData, so this might be empty from initialization in App.tsx
+		forceMeshesComp.forEach((mesh2,idx) => { //for each un-positioned 3D mesh
+		   const pos1 = frameData.forces[idx].components;
+		   const pos = frameData.forces[idx].position;
+		   if (!pos||isNaN(pos.x)||isNaN(pos.y)||isNaN(pos.z)||(pos.x===0&&pos.y===0&&pos.z===0)) {
+			  mesh2.visible = false; //hide forces with no data
+			  return;
+		   }
+		   var i = 0;
+		   while(i < 600) {
+		   
+		   const compp = pos1.y;
```

Comment:
We should also make use of the two other vector components. In the x component, some frames have a fairly significant value that is at least 1/4 of the y component, so it would be visibly noticeable and useful to see.